subjects/index.js

```diff
@@ -0,0 +1,40 @@
+const pg = require('pg');
+
+function executeSQL(context, params) {
+    const connectionString = 'postgres://postgres:postgres@localhost:5432/sample_db'
+    const client = new pg.Client({ connectionString })
+
+    client.connect();
+    const { maxLat, minLat, maxLon, minLon } = params
+
+    if (!maxLat || !minLat || !maxLon || !minLon) {
+        context.res = {
+            status: 422,
+            body: 'Query params must include maxLat, minLat, maxLon, and minLon'
+        }
+        return context.done()
+    }
+
+    const query = `SELECT * FROM subjects where lat < ${maxLat} and lat > ${minLat}

```

Comment:
We dont want to directly append a URL query param value to the SQL query like this, it leaves us open to [SQL injection](https://www.w3schools.com/sql/sql_injection.asp). Instead, use SQL parameters for handling query param inputs. 

See the bit on "Parameterized Queries" here ->
https://node-postgres.com/features/queries
(not 100% sure if youre using node-postgres here but if not, the same principals should apply and its still javascript)